examples/demo13.py

```diff
@@ -93,11 +93,11 @@ def main(argv):
     logger.debug('Read in WFIRST imaging filters.')
 
     logger.info('Reading from a parametric COSMOS catalog.')
-    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<23.5 from COSMOS.
+    # Read in a galaxy catalog - just a random subsample of 100 galaxies for F814W<25.2 from COSMOS.
```

Comment:
I think if you put back the deep=True line, you'll see what I mean.  `use_sample` gets set in COSMOSCatalog by the line:
```
full_file_name, full_image_dir, _, self.use_sample = \
            galsim.real._parse_files_dirs(file_name, image_dir, dir, noise_dir, sample)
```
This comes back with `use_sample` = None, which triggers a warning when `deep` is True.  If you didn't actually update the example file, then the logic there needs to change.